src/wwwroot/services/auth.js

```diff
@@ -214,5 +216,21 @@
         }
       });
     }
+
+    function getFreeTrialEndDate() {
+      return $http({
+        actionDescription: 'Gathering Free Trial end date',
+        method: 'GET',
+        url: RELAY_CONFIG.baseUrl + '/accounts/' + getAccountName()  + '/status/limits'
+      }).then(function (response) {
+        if (!response.data.freeTrialEndDate){
+          return null;
+        }
+        return moment(response.data.freeTrialEndDate).toDate();
```

Comment:
What do you think about to store the complete response? in that way we could use the same request and object for daily limit box in reports page.